swiftnav/src/coords/llh.rs

```diff
@@ -0,0 +1,146 @@
+use super::{Ellipsoid, ECEF, WGS84};
+use nalgebra::{ArrayStorage, Vector3};
+
+/// WGS84 geodetic coordinates (Latitude, Longitude, Height), with angles in degrees.
+///
+/// Internally stored as an array of 3 [f64](std::f64) values: latitude, longitude, and height above the ellipsoid in meters
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd, Default)]
+pub struct LLHDegrees(Vector3<f64>);
```

Comment:
Similar comments as above, it would be nice to impl some `From` traits for a few common types like `Vector3`, `[f64; 3]` and `(f64, f64, f64)`